README.md

```diff
@@ -41,17 +41,31 @@ Contributing to the handbook isn't only possible for `git` users, find steps to
 
 ### Benefits & Perks
 
+Balancing life and work:
+
 * [Flexible Holiday](benefits/flexible_holiday.md) – We trust you to take as much holiday as you need
 * [Flexible Working Hours](benefits/working_hours.md) – We are flexible with what hours you work
 * [Flexible Working Days](benefits/flexible_working.md) – We are flexible to the amount of days you work in a week
-* [Pension Scheme](benefits/pension_scheme.md) – We provide a pension scheme so you can save for your future and we'll contribute to it
-* [Friday Lunches](benefits/friday_lunch.md) – We send a random 8 of our team out to lunch every Friday so everyone can get to know each other
+* [Flexible Parental Leave](guides/welfare/parental_leave.md) – We provide flexible parental leave options
```

Comment:
We could link topics as part of a sentence:

```suggestion
* We provide [flexible leave for new parents](guides/welfare/parental_leave.md).
```

Renders as:
> We provide [flexible leave for new parents](link).

Reply:
Let's handle this in a follow up